pynamodb/attributes.py

```diff
@@ -1243,18 +1243,21 @@ def __init__(
                 raise ValueError("'of' must be a subclass of Attribute")
             self.element_type = of
 
-    def serialize(self, values):
+    def serialize(self, values, *, null_check: bool = True):
```

Comment:
`null_check` defaulting to `True` since `AttributeContainer.serialize`'s `null_check` parameter defaults to `True` too.